test/bin/ci_phase_iso_boot.sh

```diff
@@ -25,6 +25,9 @@ cd "${ROOTDIR}"
 # other scripts use virsh.
 bash -x ./scripts/devenv-builder/manage-vm.sh config
 
+# Clean up the image builder cache to free disk for virtual machines
+bash -x ./scripts/image-builder/cleanup.sh -full
```

Comment:
does it contain `bash -x`  for debugging only?

Reply:
Yes, we do this for most of the scripts we launch in CI. Otherwise, debugging requires applying special techniques.